modules/sdk/gradle-plugins-workspace/src/main/resources/schemas/client-extension.schema.json

```diff
@@ -217,6 +217,17 @@
 					"allOf": [
 						{
 							"$ref": "#/definitions/propertySets/properties/url"
+						},
+						{
+							"properties": {
+								"scriptElementAttributes": {
+									"items": {
+										"type": "object"
+									},
+									"type": "object"
```

Comment:
This doesn't seem quite right - `items` is only relevant when the schema type is `array`.

Reply:
Ops, you're right. It was an array before, we changed that but I forgot to update it.